startup_scripts/075_locations.py

```diff
@@ -17,7 +17,7 @@
         query = {field: params.pop(assoc)}
         params[assoc] = model.objects.get(**query)
 
-    rack_group, created = RackGroup.objects.get_or_create(**params)
+    location, created = Location.objects.get_or_create(**params)
 
     if created:
-        print("🎨 Created rack group", rack_group.name)
+        print("🎨 Created rack group", location.name)
```

Comment:
```suggestion
        print("🎨 Created location", location.name)
```

Should we change this?